### DISCUSSION TITLE
Run renovate in PR check

### DISCUSSION THREAD

**Discussion by ns-ggeorgiev** [renovatebot/renovate#20214] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

Hi,

I was designing in my head a pr check that makes sure that if a requirements.in file is changed it makes sure that the paired requirements.txt is regenerated.

It hit me that what I am designing is pip-compile manager. Makes me wonder if this was discussed before.

Can renovate be run in a single file mode against cloned already repo, where it can simply make the code change and thats it.

No clone, no PR, nothing else.

**Comment by viceice** [comment]
no
